src/uct/ib/base/ib_md.c

```diff
@@ -1578,13 +1578,14 @@ void uct_ib_md_parse_relaxed_order(uct_ib_md_t *md,
                                    const uct_ib_md_config_t *md_config)
 {
     if (md_config->mr_relaxed_order == UCS_CONFIG_ON) {
-        if (IBV_ACCESS_RELAXED_ORDERING) {
+        if (HAVE_DECL_IBV_ACCESS_RELAXED_ORDERING) {
             md->relaxed_order = 1;
         } else {
             ucs_warn("relaxed order memory access requested but not supported");
         }
     } else if (md_config->mr_relaxed_order == UCS_CONFIG_AUTO) {
-        if (ucs_cpu_prefer_relaxed_order()) {
+        if (HAVE_DECL_IBV_ACCESS_RELAXED_ORDERING &&
```

Comment:
can we unite this if with the if in line 1586, using && ?

Reply:
@yosefe I would rather use HAVE_DECL_IBV_ACCESS_RELAXED_ORDERING to ifdef passing the flag. It is fixed value so IMHO it cleaner to ifdef.